CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ Remember to align the itemized text with the first line of an item within a list
     Before, it was using the 0th device for the JAX-default backend.
   * A number of `jax.numpy` functions now have their arguments marked as
     positional-only, matching NumPy.
+  * `jnp.msort` is now deprecated, following the deprecation of `np.msort` in numpy 1.24.
```

Comment:
Add a sentence about when we will remove this?

Reply:
Done